src/index.js

```diff
@@ -1,35 +1,34 @@
 import './style.css';
 
-const taskObject = [
-  {
-    chore: 'wash the dishes',
-    completed: true,
-    index: 1,
-  },
+import Awesome from './modules/classAwesome.js';
 
-  {
-    chore: 'complete To Do list project',
-    completed: false,
-    index: 2,
-  },
-];
+const enter = document.querySelector('.enter');
+const tasks = document.querySelector('#tasks');
+const awesome = new Awesome();
+document.addEventListener('DOMContentLoaded', awesome.returnInfo());
```

Comment:
- [ ] [OPTIONAL] the second parameter of the `addEventListener` function takes a function. currently you are passing it undefined because `awesome.returnInfo` doesn't return anything. I think you wanted to exectur the `returnInfo` function when the dom content is loaded so you should wrap the call with an arrow function.
```suggestion
document.addEventListener('DOMContentLoaded', () => awesome.returnInfo());
```